docs/src/adoc/index.adoc

```diff
@@ -1613,47 +1620,351 @@ As in the Core API, neither `@BindMap`, `@BindBean`, `@BindFields`, nor
 
 ==== @SqlQuery
 
-SqlQuery returns a single- or multi-row result, depending on whether the method
-return type looks like a collection.
+Use the `@SqlQuery` annotation for select operations.
 
-TODO:
+Query methods may return a single- or multi-row result, depending on whether the
+method return type looks like a collection.
+
+[source,java]
+----
+public interface UserDao {
+  @SqlQuery("select name from users")
+  List<String> listNames(); // <1>
+
+  @SqlQuery("select name from users where id = ?")
+  String getName(long id); // <2> <3>
```

Comment:
Does it makes sense to mention that JDBI supports out-of-box `Optional<String>` for single row results?